docs/source/configuration.rst

```diff
@@ -88,6 +88,37 @@ in projects which utilise a lot of complicated templating.
 
 .. _templateconfig:
 
+Rule Configuration
+------------------
+
+Rules can be configured with the `.sqlfluff` config files.
+
+Common rule configurations can be set in the *[sqlfluff:rules]* section.
```

Comment:
In the code the common section is called "generic":

https://github.com/sqlfluff/sqlfluff/blob/7ffea4670c384f575dd7a59ed31da0b93ae8f800/src/sqlfluff/core/rules/base.py#L951-L954

But I'm leaning towards "common" as the user-facing term to use. Please advise :)

Reply:
It’s called common in the default config too:
https://github.com/sqlfluff/sqlfluff/blob/7ffea4670c384f575dd7a59ed31da0b93ae8f800/src/sqlfluff/core/default_config.cfg#L37-L38